File: src/Shared/Logistics.Shared.Models/TruckGeolocationDto.cs
namespace Logistics.Shared.Models;

public class TruckGeolocationDto
{
    public required string TruckId { get; set; }
    public required string TenantId { get; set; }
    public required double Latitude { get; set; }
    public required double Longitude { get; set; }
    public string? CurrentAddress { get; set; }
    public string? TruckNumber { get; set; }
    public string? DriversName { get; set; }
}

File: src/Client/Logistics.OfficeApp/src/app/core/models/truck/TruckGeolocationDto.ts
import {AddressDto} from "../AddressDto";

export interface TruckGeolocationDto {
  truckId: string;
  tenantId?: string;
  latitude: number;
  longitude: number;
  currentAddress?: AddressDto;
  truckNumber?: string;
  driversName?: string;
}
